index.js

```diff
@@ -1,95 +1,88 @@
-const path          = require('path');
-const EmailTemplate = require('email-templates').EmailTemplate;
-const _             = require('underscore');
-const waterfall     = require('async/waterfall');
+const _ = require('underscore')
```

Comment:
an `'use strict'`  at the top would be nice

Reply:
It should be in there, see https://www.w3schools.com/js/js_strict.asp.

Reply:
done but that link doesnt really explain why (i mean why you like use strict?) - cos it's a debate we programmers have